src/components/structures/RoomView.js

```diff
@@ -360,6 +362,18 @@ module.exports = React.createClass({
                 });
             }
         }
+
+        // update ther tab complete list as it depends on who most recently spoke,
+        // and that has probably just changed
+        if (ev.sender) {
+            this.tabComplete.onMemberSpoke(ev.sender);
+
+            const myUserId = MatrixClientPeg.get().credentials.userId;
+            const members = this.state.room.getJoinedMembers().filter(function(member) {
+                if (member.userId !== myUserId) return true;
```

Comment:
https://github.com/matrix-org/matrix-react-sdk/pull/341/files#r70994084

Also - is there any place where we **don't** do this prior to passing it to `UserProvider.getInstance().setUserList`? Would it make more sense for `setUserList` to do this operation?